templates/submission/status.html

```diff
@@ -52,6 +53,17 @@
         });
         </script>
     {% endif %}
+    {% if not submission.is_graded %}
+        <script type="text/javascript">
+            $(function () {
+                $(document).keydown(function(ev) {
```

Comment:
```suggestion
                $(document).keydown(function(e) {
```
We use `e` for events everywhere else, unless there's another reason?

Reply:
This JS was copied from somewhere else in this repository, but I can't seem to find it anymore. Will change to `e`.